#include <string.h> missing for ffsll

I have compiled rob-twophase using the provided makefile on Ubuntu using GCC 5.4.0.

During compilation I get this error mesage:
```
src/coord.cpp: In function ‘void coord::init_move(uint16_t (*)[18], int, int (*)(const cubie::cube&), void (*)(cubie::cube&, int), void (*)(const cubie::cube&, const cubie::cube&, cubie::cube&), bool)’:
src/coord.cpp:233:30: error: ‘ffsll’ was not declared in this scope
           int m = ffsll(moves) - 1;
```

ffs is defined in the header strings.h, while ffsll is defined in the header string.h:
https://man7.org/linux/man-pages/man3/ffsll.3.html
However, only strings.h is included.

This applies to coord.cpp, prun.cpp and solve.cpp as they all use ffsll.
If I add #include <string.h> to these files, it compiles without errors.

--- a/src/coord.cpp
+++ b/src/coord.cpp
@@ -3,6 +3,7 @@
 #include <algorithm>
 #include <bitset>
 #include <strings.h>
+#include <string.h>
 
 #include "cubie.h"
 

--- a/src/prun.cpp
+++ b/src/prun.cpp
@@ -3,6 +3,7 @@
 #include <bitset>
 #include <iostream>
 #include <strings.h>
+#include <string.h>
 
 namespace prun {
 

--- a/src/solve.cpp
+++ b/src/solve.cpp
@@ -2,6 +2,7 @@
 
 #include <algorithm>
 #include <strings.h>
+#include <string.h>
 #include <thread>
 #include "prun.h"
 #include "sym.h"

